src/main/java/org/jenkinsci/plugins/workflow/flow/FlowDefinition.java

```diff
@@ -76,4 +79,8 @@ public FlowExecution create(FlowExecutionOwner handle, List<? extends Action> ac
         return (FlowDefinitionDescriptor) super.getDescriptor();
     }
 
+    public Collection<? extends SCM> getSCMs() {
```

Comment:
For a new public method, I'd like to see a Javadoc comment explaining the method, what it does and doesn't do, etc. In this case, I think it would be good to clarify the behavior with something along these lines:

```
/**
 * Returns a list of all {@link SCM SCMs} that are part of the static configuration of the {@link FlowDefinition}.
 * For example, this method returns the SCM used to retrieve the Jenkinsfile for {@code CpsScmFlowDefinition}.
 * Does not include any SCMs used dynamically during Pipeline execution. 
 * May be empty if the Pipeline does not include any statically configured SCMs.
 */
```